src/c-writer.cc

```diff
@@ -2195,6 +2236,19 @@ void CWriter::Write(const LoadSplatExpr& expr) {
   PushType(result_type);
 }
 
+void CWriter::Write(const LoadZeroExpr& expr) {
+  assert(module_->memories.size() == 1);
+  Memory* memory = module_->memories[0];
+  Type result_type = expr.opcode.GetResultType();
+  Write(StackVar(0, result_type), " = ", expr.opcode.GetName(), "(",
+        ExternalPtr(memory->name), ", (u64)(", StackVar(0), ")");
+  if (expr.offset != 0)
+    Write(" + ", expr.offset, "u");
+  Write(");", Newline());
```

Comment:
this differs from the one above it, which includes the offset in the u64 cast. I guess the result is the same, but it would be good to be consistent.